wordpress-mobile/WordPress-Android
# Feature/discover sorting UI components

## Added Ui components for Improvement: Reader : Sorting option for Discover or topic

### To test:
- enable FilterDiscoverFeatureConfig under Features in development
- go to reader > discover 
- check the feature is working properly

| Initial Condition | On Clicking Filter button | On Scrolling down |
|---|---|---|
|![scrolling down](https://user-images.githubusercontent.com/46663626/109260083-e40e4780-782b-11eb-992b-a122a16a8e46.jpg)|![sorting option dialog](https://user-images.githubusercontent.com/46663626/109260086-e670a180-782b-11eb-9782-7af5da5311ee.jpg)|![initial condition](https://user-images.githubusercontent.com/46663626/109260087-e7093800-782b-11eb-83d5-b7a541062c70.jpg)|

### PR submission checklist:
